pkg/utils/common/controller.go

```diff
@@ -672,3 +674,34 @@ func (s *SuiteController) DeleteAllPipelineRunsAndTasks(namespace string) error
 
 	return nil
 }
+
+// CreateRegistryAuthSecretKeyVlaue creating key/value kubernetes secret.
+func (s *SuiteController) CreateRegistryAuthSecretKeyValueDecoded(secretName, namespace, secretStringData, secretData string) (*corev1.Secret, error) {
```

Comment:
Maybe I'm missing something, but why is the function called `*RegistryAuthSecret*`? This secret is for pyxis key+cert, not for a registry, right?